lms/static/js/spec/student_account/login_spec.js

```diff
@@ -208,6 +210,20 @@
                     expect(thirdPartyAuthView).not.toContain($('form-field'));
                 });
 
+                it('does not display the enterprise login button', function() {
```

Comment:
Thank you for adding this test, it would be great if we add a test for the enabled case as well.

Reply:
Thanks for your review and comments, I'll add the proposed changes.